ui/app/templates/vault/cluster/redirect.hbs

```diff
@@ -0,0 +1 @@
+<LogoSplash />
```

Comment:
Is this the reason for having a new Route? So the UI can update straight away after it leaves the auth route?

Reply:
This template should actually never show up since everything happens in the `beforeModel` hook but I thought it prudent to have it just in case